.Documentation/articles/innoactive-creator/course-creation-wizard.md

```diff
@@ -2,13 +2,13 @@
 
 ## Open the Wizard
 
-In order to open the wizard, go to the Unity's toolbar and select: `Innoactive` > `Creator` > `Create New Course...`.
+In order to open the wizard, go to the Unity's toolbar and select: `Innoactive` > `Create New Course...`.
 
 [![Create New Option](../images/course-creation-wizard/create-new.png "")](../images/course-creation-wizard/create-new.png)
 
 ## Create a New Training Course
 
-In the `Training Course Wizard` window you can set the `Training Course Name` for a new training course. It will also be the filename. You can change the name later in the [Workflow Editor](workflow-editor.md).
+In the `Training Course Wizard` window you can set the `Training Course Name` for a new training course. It will also be the filename. You can change the name later in the [Workflow window](workflow-editor.md).
```

Comment:
```suggestion
In the `Training Course Wizard` window you can set the `Training Course Name` for a new training course. It will also be the filename. You can change the name later in [the Workflow window](workflow-editor.md).
```

Reply:
Why do you want to add 'the' to the link?